src/Lib.hs

```diff
@@ -291,14 +283,28 @@ inspectAttachment ticketInfo@TicketInfo{..} att = do
                         , zrIsPublic    = cfgIsCommentPublic
                         }
 
+responseNoLogs :: TicketInfo -> App ZendeskResponse
+responseNoLogs TicketInfo{..} = do
+    Config {..} <- ask
+    pure ZendeskResponse
+             { zrTicketId = ticketId
+             , zrComment  = prettyFormatNoLogs
+             , zrTags     = [renderTicketStatus NoLogAttached]
+             , zrIsPublic = cfgIsCommentPublic
+             }
+
 -- | Filter analyzed tickets
 filterAnalyzedTickets :: [TicketInfo] -> [TicketInfo]
 filterAnalyzedTickets ticketsInfo =
     filter ticketsFilter ticketsInfo
   where
     ticketsFilter :: TicketInfo -> Bool
-    ticketsFilter ticketInfo =
-        isTicketAnalyzed ticketInfo && isTicketOpen ticketInfo && isTicketBlacklisted ticketInfo
+    ticketsFilter ticketInfo = 
+        all (\predicate -> predicate ticketInfo) [ isTicketAnalyzed
```

Comment:
A complicated way to say a simple thing. Remember that you will have to maintain this code and that doing something smart may not be the best things to do - do something simple instead.
I know it seems kind of harsh to say that, but it's important that you understand how little changes like these make the code more complex. I actually have to think about this, which is something I like to avoid as much as possible.
And if this builds, isn't `all ticketInfo [ ... ]`  an option?